src/lib/shell/commands/Dns.cpp

```diff
@@ -51,7 +51,8 @@ class DnsShellResolverDelegate : public Dnssd::DiscoverNodeDelegate, public Addr
         result.address.ToString(addr_string);
 
         streamer_printf(streamer_get(), "Resolve completed: %s\r\n", addr_string);
-        streamer_printf(streamer_get(), "   Supports TCP:                  %s\r\n", result.supportsTcp ? "YES" : "NO");
+        streamer_printf(streamer_get(), "   Supports Client TCP:  %s\r\n", result.supportsTcpClient ? "YES" : "NO");
+        streamer_printf(streamer_get(), "   Supports Server TCP:  %s\r\n", result.supportsTcpServer ? "YES" : "NO");
```

Comment:
Why are these talking about "Client TCP" and "Server TCP" while other logs talk about "TCP Client" and "TCP Server"?  Why the inconsistency?

Reply:
good point, fixed